cmd/ci-secret-bootstrap/secretstats.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/montanaflynn/stats"
+
+	coreapi "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+type secretStats struct {
+	count  int
+	median float64
+}
+
+func generateSecretStats(secretsByClusterAndName map[string]map[types.NamespacedName]coreapi.Secret) (secretStats, error) {
+	if len(secretsByClusterAndName) == 0 {
+		return secretStats{}, nil
+	}
+	count := 0
+	var size []float64
+	for _, secretMap := range secretsByClusterAndName {
+		for _, secret := range secretMap {
+			count = count + 1
```

Comment:
`count += len(secretMap)` (outside the loop).

Reply:
The names (here and in the resulting struct) could also be more explicit about what it is that is being counted and measured.

Reply:
Added GoLang docs for the fields.